src/main/java/redis/clients/jedis/ScanResult.java

```diff
@@ -17,14 +17,31 @@ public ScanResult(byte[] cursor, List<T> results) {
     this.results = results;
   }
 
+  /**
+   * Returns the new value of the cursor
+   * @return the new cursor value. {@link ScanParams#SCAN_POINTER_START} when a complete iteration has finished
+   */
   public String getCursor() {
     return SafeEncoder.encode(cursor);
   }
 
+  /**
+   * Is the iteration complete. I.e. was the complete dataset scanned.
+   *
+   * @return true if the iteration is complete
+   */
+  public boolean isCompleteIteration() {
```

Comment:
You need to clone redis from the official repo https://github.com/antirez/redis run `make` and `make install` to install the latest version.